### PR TITLE
creat the files

### PR DESCRIPTION
**Pull request by Sanyyouisf**

### CODE REVIEW

**Comment by Sanyyouisf** [comment]
@drdreed
I have some question for my  here:
1- i don't know why the function on line 7 in getjson.js file doesn't work.
2- the delete button in the second part( the second json file) doesn't fire , instead of  it is the same as the delete button in the  first part .
3- whey i should add this line 
$(document).ready(function(){ 
before every jquery function to make it work , it suppose to add it only one time at the beginning of the js file . 
thanks .

**Comment by Sanyyouisf** [comment]
@drdreed
ready to merge now

**Comment by Sanyyouisf** [comment]
@drdreed
waiting for approval to merge

File: getjson.js
Comment (DevDreed) [inline getjson.js]: Please remove unused code.

**Review by DevDreed — CHANGES_REQUESTED** [review]
Please remove unused/commented out code.

**Review by DevDreed — APPROVED** [review]
👍

**Comment by Sanyyouisf** [comment]
@drdreed
thanks

**Pull request merged**
